src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -119,6 +131,9 @@ export default Relay.createContainer(Message, {
           content_type
           download_url
           file_name
+          id
+          ${ImagePreview.getFragment("imageAttachment")}
+          ${PDFPreview.getFragment("pdfAttachment")}
```

Comment:
You weren’t letting Relay handle the fetching of the data for these component, but were fetching it yourself and then pushing that into components that Relay expected to manage, hence the warning.